test/test-lib-postgresql.sh

```diff
@@ -37,4 +40,16 @@ function test_postgresql_imagestream() {
   ct_os_test_image_stream_template "${THISDIR}/imagestreams/postgresql-${OS%[0-9]*}.json" "${THISDIR}/examples/postgresql-ephemeral-template.json" postgresql "-p POSTGRESQL_VERSION=${VERSION}${tag}"
 }
 
+
+function run_latest_imagestreams_test() {
+  local result=1
+  # Switch to root directory of a container
+  echo "Testing the latest version in imagestreams"
+  pushd "${THISDIR}/.." >/dev/null || return 1
+  ct_check_latest_imagestreams
+  result=$?
+  popd >/dev/null || return 1
+  return $result
+}
+
```

Comment:
The `test_postgresql_imagestream` still fails.
Log:
```
Warning: unknown field "spec.template.spec.containers[0].capabilities"
deploymentconfig.apps.openshift.io/postgresql created
.
.
.
View details with 'oc describe <resource>/<name>' or list resources with 'oc get all'.
Build config bc/ does not exist for some reason.
Import probably failed.
project.project.openshift.io "sclorg-test-28907" deleted
Project sclorg-test-28907 was deleted properly
Test for image 'rhel8/postgresql-15:1' FAILED (exit code: 1)
```

Reply:
This should be fixed by https://github.com/sclorg/postgresql-container/pull/499/commits/47aa17220c4214ae7f652d6ac9a97d90f933356a. We do not have yet registry.redhat.io/rhel9/postgresql-15 image yet. The same for RHEL8. postgresql-15 image is not yet generally available.

Reply:
Should we re-run the tests then?

Reply:
Already done.